src/LiveComponent/CHANGELOG.md

```diff
@@ -2,6 +2,49 @@
 
 ## 2.14.0
 
+-   [BC BREAK] The `data-action-name` attribute behavior was removed in favor of
+    using Stimulus "action parameters" and `data-live-action-param`. This is a
+    breaking change if you were using the `data-action-name` attribute directly
+    in your templates.
+
+    To upgrade your application, follow these changes:
+
+    ```diff
+    <button
+        data-action="live#action"
+    -    data-action-name="debounce(300)|save"
+    +    data-live-action-param="debounce(300)|save"
+    >Save</button>
+    ```
+
+    To pass arguments to an action, also use the Stimulus "action parameters" syntax:
+
+    ```diff
+    <button
+        data-action="live#action"
+    -     data-action-name="addItem(id={{ item.id }}, itemName=CustomItem)"
+    +     data-live-action-param="addItem"
+    +     data-live-id-param="{{ item.id }}"
+    +     data-live-item-name-param="CustomItem"
+    >Add Item</button>
+    ```
```

Comment:
Maybe I am missing something but it looks like by doing this change we losing de debounce. Maybe we should also show a good way to keep the debounce?

Reply:
Thanks - that was accidental. The diff should still include the `data-live-action-param="debounce|(300)addItem". I'll fix that when I figure out the direction for the PR :)